### PR TITLE
Modify the SshV0 task to direct the output to process.stdout instead of the console.log

### PR DESCRIPTION
**Pull request by ivanduplenskikh**
**Task name**: SshV0

**Description**: This PR changes redirection from console.log to process.stdout in the SshV0 task, so console.log adds newlines at the end of lines.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** #20472 

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Comment by ivanduplenskikh** [comment]
Closing this PR since the fix doesn't need to be implemented. Details in the linked issue.

**Review by DmitriiBobreshev — APPROVED** [review]

**Review by KonstantinTyukalov — APPROVED** [review]

File: Tasks/SshV0/ssh2helpers.ts [hunk 1]
```diff
@@ -147,7 +147,11 @@ export function runCommandOnRemoteMachine(
             }).on('data', (data) => {
                 if (data) {
                     // "data" can be a buffer. Format it here so it outputs as a string
-                    console.log(data.toString('utf8'));
+                    if (tl.getPipelineFeature("redirectTaskOutputToProcessStdout")) {
```
Comment (KonstantinTyukalov) [inline Tasks/SshV0/ssh2helpers.ts:150]: nit: please add the task name to the flag name for clarity

Comment (KonstantinTyukalov) [reply Tasks/SshV0/ssh2helpers.ts:150]: do you mean that this flag will be reused in multiple tasks? In this case, it's ok for me to keep the current name 🙂

**Review by max-zaytsev — APPROVED** [review]

File: Tasks/SshV0/ssh2helpers.ts (hunk 1 above)
Comment (ivanduplenskikh) [reply Tasks/SshV0/ssh2helpers.ts:150]: I have found a few more [buffer conversions](https://github.com/search?q=repo%3Amicrosoft%2Fazure-pipelines-tasks+%22data.toString%28%27utf8%27%29%22+path%3A%2F%5ETasks%5C%2F%2F&type=code).
The solution might also be applicable in these scenarios.

**Review by DergachevE — APPROVED** [review]

**Comment by avishnyakov** [comment]
This is awesome. Happy to test new version `0.247.1` - let me now how we can do this.

**Pull request merged**